AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -1117,23 +1115,7 @@ - (ASScrollDirection)_scrollDirectionForVelocity:(CGPoint)scrollVelocity
 
 - (ASScrollDirection)scrollableDirections
 {
-  //Cache results of layoutInspector to ensure flags are up to date if getter lazily loads a new one.
-  id<ASCollectionViewLayoutInspecting> layoutInspector = self.layoutInspector;
-  if (_layoutInspectorFlags.scrollableDirections) {
-    return [layoutInspector scrollableDirections];
-  } else {
```

Comment:
My main question about this diff is whether this logic below, in the Else clause, should live somewhere — either in the ASCollectionViewLayoutInspector base class, or in the Flow layout version.  Or is it really no longer needed?